### PR TITLE
Add a blogpost about the label-prop-app

### PR DESCRIPTION
**Pull request by surchs**
I added a blogpost for the [label-props app](https://dash-gallery.plotly.host/dash-label-properties/).
The site rendered well on my end.

Open points:

- [ ] are we showcasing all the important features?
- [ ] should we mention something else in the "next" section
- [ ] is there some way to embed an html rendering of a px figure to make it more exciting?

### CODE REVIEW

File: _posts/2020-11-10-region-properties-exploration.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title:  "Combine px.imshow and datatables to explore image segmentation"
+author:	Sebastian Urchs and Emmanuelle Gouillart
+---
+
+![Screenshot of region properties app](/assets/region-properties-app.png)
+
+Image processing workflows generate multimodal information that need to be explored and interpreted. For example, with skimage we can build a lightweight image segmentation workflow and compute properties of the segmented regions in the image. We may want to look at the segmented regions in an interactive visualization such as px.imshow, but we may want to look at the computed region properties in a table where we can sort and search values. Here we have built a dash app that combines the best of both worlds by interactively linking visual and tabular data.
```
Comment (emmanuelle) [inline _posts/2020-11-10-region-properties-exploration.md]:
```suggestion
Image processing workflows generate multimodal information that need to be explored and interpreted. For example, with scikit-image we can build a lightweight image segmentation workflow and compute properties of the segmented regions in the image. We may want to look at the segmented regions in an interactive visualization such as plotly's`px.imshow`, but we may want to look at the computed region properties in a table where we can sort and search values. Here we have built a Dash app that combines the best of both worlds by interactively linking visual and tabular data.
```

File: _posts/2020-11-10-region-properties-exploration.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+Notice that we have asked `px.imshow` to compress the image into a bytestring with the `binary_string=True` argument in order to keep the amount of data being sent over the network small. 
+
+For the same reason we have choosen here to compute the contours of the labeled regions with scikit-image and then add them as Scatter traces with custom hoverinfo to the figure. This approach is very efficient because:
+1. We do the computation of the region contours on the server and in python where it is fast
+2. For each region we only need to send very little information over the network:
+    - the points of the computed contour trace
+    - a single string with the hover info
+
+We could have achieved a similar outcome in fewer lines of code by using the [plotly Contour traces](https://plotly.com/python/reference/contour/#contour). The downside of this approach is, that not only do we need to send the full labeled image array over the network, we also ask the browser to compute the contours of this labeled array for us. This can be a good solution for a local application but it can become slow quite quickly in a dash app.
```
Comment (emmanuelle) [inline _posts/2020-11-10-region-properties-exploration.md]:
```suggestion
We could have achieved a similar outcome in fewer lines of code by using the [plotly Contour traces](https://plotly.com/python/reference/contour/#contour). The downside of this approach is, that not only do we need to send the full labeled image array over the network, we also ask the browser to compute the contours of this labeled array for us. While this solution is acceptable for medium-sized arrays, we get a better performance with our solution for large image arrays and deployed apps.
```

Comment (emmanuelle) [reply _posts/2020-11-10-region-properties-exploration.md]: trying to talk about limitations in a positive and diplomatic way ;-)

**Review by emmanuelle — APPROVED** [review]
Thanks! You really nailed the gist of the app. I just left two small comments, feel free to merge when you want :woman_dancing:

**Comment by emmanuelle** [comment]
For the record, it's possible to embed raw html (including plotly/px figures) in markdown.

**Comment by surchs** [comment]
> For the record, it's possible to embed raw html (including plotly/px figures) in markdown.

Oh wow. I didn't realize I could just literally dump the html content of the figure inside the markdown file. That is very nice. Thanks for the hint!

**Comment by surchs** [comment]
Thanks for the improvements @emmanuelle. Currently the blogpost is backdated to 10/11/2020. Is there a specific day this should go online?

**Pull request merged**